index.bs

```diff
@@ -34,6 +34,7 @@ spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
     urlPrefix: dom.html
         type: interface
             text: Document; url: document
+        type:event; text:close
```

Comment:
What's this for?

Reply:
This fixes a compile error I got, not sure why it cropped up now but it couldn't distinguish between this close and one in serviceworker

Reply:
Well, that should go into `<pre class=link-defaults>` not `<pre class=anchors>`... but it exists in main, let me see what's up...

Reply:
Okay, this is due to linking weirdness elsewhere - it should be a reference internal to the spec. I'll put up a fix.

Reply:
https://github.com/w3c/IndexedDB/pull/384